packages/web-app-files/src/store/actions.ts

```diff
@@ -522,6 +523,40 @@ export default {
     if (preview) {
       commit('UPDATE_RESOURCE_FIELD', { id: resource.id, field: type, value: preview })
     }
+  },
+
+  loadAncestorMetaData({ commit, state }, { path, space, client }) {
+    const ancestorMetaData = {}
+    const promises = []
+    const parentPaths = getParentPaths(path, true)
```

Comment:
```suggestion
    const parentPaths = getParentPaths(path, false)
```

... since we only want to propfind parents, right?

Reply:
As discussed: Let's keep the current folder because we need it anyways.